cpu/stm32_common/periph/timer.c

```diff
@@ -74,7 +74,27 @@ int timer_set_absolute(tim_t tim, int channel, unsigned int value)
         return -1;
     }
 
+#if defined(CPU_MODEL_STM32L412KB)
```

Comment:
*When in Rome, do as the Roman's do.*

I don't like editing vendor files, but having this be the only member of that family that *didn't* edit that part of the vendor files makes it odd too.

I don't see a generic way to detect this either, so let's just follow the precedent and also edit `stm32l412xx.h` to change `CCR1…4` -> `CCR[4]`.

Then the changes to common files should be minimal and this will be good to merge.

Reply:
I agree with you @benpicco. It's not nice to edit CMSIS files but since this is done like this for others...